### ISSUE TITLE
Label string too long on instance.start cmd

### ISSUE THREAD

**Issue by ArangoGutierrez** [apptainer/singularity#1185]
## Version of Singularity:

2.4-master.g718360bb

When doing the  [example](https://github.com/singularityware/singularityware.github.io/blob/master/pages/docs/user-docs/docs-instance-start.md)

### Expected behavior
```bash
$ singularity instance.start shub://GodloveD/lolcow cow1
```
### Actual behavior

```bash
$ singularity instance.start shub://GodloveD/lolcow cow1
ERROR  : Label string too long
ABORT  : Retval = 255
ERROR  : Daemon failed to start
ABORT  : Retval = -1
```

Debug + VERBOSE output attached:
[instance_start_issue.txt](https://github.com/singularityware/singularity/files/1519511/instance_start_issue.txt)

**Comment by GodloveD** [comment]
I can't seem to replicate this.  Can you see if it is still an issue with the latest release-2.4 branch?

**Comment by ArangoGutierrez** [comment]
Works! on latest release-2.4 branch thanks @GodloveD

**Comment by GodloveD** [comment]
Great!  😸

**Comment by ArangoGutierrez** [comment]
Error is back on 
```bash
$ singularity --version
2.4.1-release-2.4.g56581cf3
```

**Comment by GodloveD** [comment]
Crap!  💩

**Comment by yarikoptic** [comment]
indeed it is so even in 2018
```shell
$> singularity instance.start shub://GodloveD/lolcow cow1
Progress |===================================| 100.0% 
ERROR  : Label string too long
ABORT  : Retval = 255
ERROR  : Daemon failed to start
ABORT  : Retval = -1

$> singularity --version
2.4.5-dist
```

**Comment by yarikoptic** [comment]
and even with 2.4.6 (just uploaded to Debian and NeuroDebian)

**Comment by jmstover** [comment]
Hi @yarikoptic .  Have a PR in which should fix this issue. At least locally, I'm not hitting it with what I've tested.

**Comment by jscook2345** [comment]
@ArangoGutierrez @yarikoptic 

This looks to be fixed in 3.0.0. Please re-open if the issue persists and we'll work from there. Thanks!

**Comment by yarikoptic** [comment]
Thanks!
